assets/mkEnv/env-from-lockfile.nix

```diff
@@ -0,0 +1,26 @@
+# Build an environment from a collection of packages
+{
+  lockfilePath ?
+    throw
+    "flox: You must provide the path to a lockfile.",
+  system ? builtins.currentSystem or "unknown",
```

Comment:
suggestion nonblocking: would it be better to error instead of defaulting to `unknown`?

Reply:
If I understand correctly this will only be "unknown" if `builtins.currentSystem` is somehow `false`. Seems unlikely and this is also throwaway code.